Example-Crypto-REXX/RXCRYPT1.rexx

```diff
@@ -0,0 +1,2431 @@
+/*         target name and status info for LPARs                       */
+/*    Call Get LPAR Properties to retrieve a set of specific           */
+/*         Image Activation Profile attributes for LPARs               */
+/*         in the specified status (default = operating)               */
+/*         Sample assumes the image activation profile name is         */
+/*         the same as the LPAR name                                   */
+/*    Write collected content out to the provided partitioned data set */
+/*                                                                     */
+/* INVOCATION:                                                         */
+/*     RXCRYPT1 -D outputDataSet [-C CPCname] [-S Status] [-I] [-V]    */
+/*                                                                     */
+/*     Required input parameters:                                      */
+/*        -D <data set name> is the name of a pre-existing partitioned */
+/*           data set, if the LPAR query is successful, a member       */
+/*           containing the query information in CSV format will be    */
+/*           stored into the data set, the member will either be named */
+/*           LOCAL or the CPC name provided via -C option              */
+/*     Optional input parameters:                                      */
+/*        -C <CPCname> name of the CPC to query, default is LOCAL CPC  */
+/*        -S <Status> LPAR Status of the profiles to query,            */
+/*           operating - query profiles for active/operating images    */
+/*           not-operating - query profiles for active/not operating   */
+/*                           images                                    */
+/*           not-activated - query profiles for not activated images   */
+/*           Any other value, or no parm specified will default to     */
+/*           operating                                                 */
+/*        -I indicate running out of ISV REXX environment,             */
+/*           default is TSO/E                                          */
+/*        -V turn on additional verbose JSON tracing                   */
+/*                                                                     */
+/*        EX 'HWI.HWIREST.REXX(RXCRYPT1))'                             */
+/*            '-D HWI.CRYPTO -I'                                       */
+/*            data set is HWI.CRYPTO, run out of ISV REXX,             */
+/*            if successful will create HWI.CRYPTO(LOCAL)              */
+/*                                                                     */
+/*        EX 'HWI.HWIREST.REXX(RXCRYPT1))'                             */
+/*            '-D HWI.CRYPTO -S not-activated'                         */
+/*            data set is HWI.CRYPTO, run from TSO REXX,               */
+/*            query crypto attributes for not-activated LPARs          */
+/*            if successful will create HWI.CRYPTO(LOCAL)              */
```

Comment:
make updates similar to what will be done for the README

Reply:
Done